README.md

```diff
@@ -204,6 +209,10 @@ sorting with Corfu:
   and `M-s` is bound to `prescient-toggle-map` while the Corfu pop-up
   is active.
 
+  If disabled, see also the user options `completion-styles` and
+  `completion-category-overrides` and the variable
+  `completion-category-defaults`, which would have been configured.
```

Comment:
Could you be a bit more explicit here? Mention explicitly that these variables are modified, such that the `prescient` completion style is added to the list. Usually packages/modes don't modify these variables, in particular not `completion-styles`.

Reply:
I've tried to be clearer. How is it now?

Reply:
Looks good.

Reply:
Looks good.